src/qt/transactionfilterproxy.cpp

```diff
@@ -8,6 +8,7 @@
 #include <qt/transactionrecord.h>
 
 #include <cstdlib>
+#include <limits>
```

Comment:
This change is unrelated after the recent rebase and could be dropped.

Reply:
Replaced it with the headers that are actually used.